functions/pupil-status/index.js

```diff
@@ -24,7 +22,7 @@ module.exports = async function (context, pupilStatusMessage) {
   // Store the raw message to an audit log
   context.bindings.pupilEventsTable = []
   const entity = {
-      // TODO will we always have a check code?
+    // TODO will we always have a check code?

```

Comment:
i asked myself this question yesterday when looking at the message schemas.  it will no doubt emerge in the next couple of weeks